### PR TITLE
[ci-visibility] Custom commands with support for more CI providers

### PR DESCRIPTION
**Pull request by rodrigo-roca**
### What does this PR do? What is the motivation?
We are expanding the support of the CI Visibility custom commands feature to more CI providers. These CI providers are:

- Github Actions
- Gitlab
- Azure pipelines
- AWS Codepipeline
- Buildkite

We previously only supported CircleCI and Jenkins

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 1]
```diff
@@ -123,6 +117,18 @@ The AWS CodePipeline integration supports correlating **CodeBuild** actions with
 
 <div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a></div>
 
+### Add the pipeline execution ID as an environment variable
+
+The pipeline execution ID is an identifier Datadog needs to uniquely identify a pipeline. This is needed for correlating pipelines with tests, custom tags and custom commands:
```
[the remark this answers is not in the document]
Comment (rodrigo-roca) [reply content/en/continuous_integration/pipelines/awscodepipeline.md:122]: Right, good point. I will change it to `pipeline execution`

File: content/en/continuous_integration/pipelines/custom_commands.md [hunk 2]
```diff
@@ -51,7 +56,7 @@ echo "Hello World"
 </code>
 </pre>
 {{< /site-region >}}
-{{< site-region region="us5,gov" >}}
+{{< site-region region="gov" >}}
```
[the remark this answers is not in the document]
Comment (rodrigo-roca) [reply content/en/continuous_integration/pipelines/custom_commands.md:59]: As you saw in the backend guild we forgot to remove it. Wanted to confirm haha

**Review by rodrimendoza — APPROVED** [review]

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 3]
```diff
@@ -104,14 +105,7 @@ The event pattern above sets up the integration only for the `first-pipeline` an
...
 If you are using [Test Visibility][8] and your pipeline contains one or more [AWS CodeBuild][9] actions to execute tests, you can correlate your tests
-with the related pipeline inside Datadog Pipeline Visibility.
-1. In the AWS Console, go to your pipeline configuration and click **Edit**
-2. Go to the stage containing the AWS CodeBuild action, click **Edit Stage**, and then edit the relevant action.
-3. Under **Environment variables**, add an environment variable.
-Name the variable `DD_PIPELINE_EXECUTION_ID`, and the value `#{codepipeline.PipelineExecutionId}`. Leave the type as _Plaintext_.
-4. Click **Done** to save your changes.
-
-The steps above allow you to add the pipeline execution ID to your CodeBuild action environment variables. For more information on working with variables, see the [official AWS guide][10].
+with the related pipeline inside Datadog Pipeline Visibility. To do this [refer to the section below](#add-the-pipeline-execution-id-as-an-environment-variable)
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:108]:
```suggestion
with the related pipeline inside Datadog Pipeline Visibility. For instructions, refer to [Add the pipeline execution ID](#add-the-pipeline-execution-id-as-an-environment-variable).
```

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 4]
```diff
@@ -123,6 +117,18 @@ The AWS CodePipeline integration supports correlating **CodeBuild** actions with
 
 <div class="alert alert-info"><strong>Note</strong>: Job log collection is not available for <a href="https://docs.datadoghq.com/data_security/pci_compliance/?tab=logmanagement">PCI-compliant organizations.</a></div>
 
+### Add the pipeline execution ID as an environment variable
+
+The pipeline execution ID is an identifier Datadog needs to uniquely identify a pipeline execution. This is needed for correlating pipelines with tests and custom commands:
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:122]:
```suggestion
The pipeline execution ID is an identifier Datadog needs to uniquely identify a pipeline execution. Perform the following steps to assign a pipeline execution ID to correlate pipelines with tests and custom commands:
```

File: content/en/continuous_integration/pipelines/custom_commands.md [hunk 5]
```diff
@@ -20,8 +20,13 @@ Custom commands provide a way to trace individual commands in your CI pipelines,
 
 Custom commands work with the following CI providers:
 
+- GitHub.com (SaaS) with datadog-ci CLI >= 2.40. For sending custom commands in GitHub Actions [see section below](#trace-a-command-in-github-actions)
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_commands.md:23]:
```suggestion
- GitHub.com (SaaS) with datadog-ci CLI >= 2.40. For sending custom commands in GitHub Actions, see [Trace a command](#trace-a-command-in-github-actions).
```

File: content/en/continuous_integration/pipelines/custom_commands.md [hunk 6]
```diff
@@ -20,8 +20,13 @@ Custom commands provide a way to trace individual commands in your CI pipelines,
...
+- GitLab (SaaS or self-hosted >= 14.1) with datadog-ci CLI >= 2.40.
 - Jenkins with the Datadog plugin >= v3.2.0
 - CircleCI
+- Azure DevOps Pipelines with datadog-ci CLI >= 2.40.
+- AWS Codepipeline with datadog-ci CLI >= 2.40. Follow [these instructions][6] to set up custom commands in AWS Codepipeline.
```
Comment (urseberry) [inline content/en/continuous_integration/pipelines/custom_commands.md:28]:
```suggestion
- AWS Codepipeline with datadog-ci CLI >= 2.40. Follow [Adding custom commands][6] to set up custom commands in AWS Codepipeline.
```

**Review by urseberry — APPROVED** [review]

**Pull request merged**